diff --git a/setup.py b/setup.py
index 7d54d90..9ceacd6 100644
--- a/setup.py
+++ b/setup.py
@@ -2,7 +2,6 @@ import re
 from io import open
 from pathlib import Path
 
-import requests
 from setuptools import find_packages, setup
 
 this_directory = Path(__file__).parent
@@ -14,24 +13,6 @@ with open(str(this_directory / "itables/version.py")) as f:
     version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]", version_file, re.M)
     version = version_match.group(1)
 
-external = Path(__file__).parent / "itables" / "external"
-if not external.is_dir():
-    external.mkdir()
-for name, url in [
-    ("jquery.min.js", "https://code.jquery.com/jquery-3.6.0.min.js"),
-    (
-        "jquery.dataTables.min.css",
-        "https://cdn.datatables.net/1.13.1/css/jquery.dataTables.min.css",
-    ),
-    (
-        "jquery.dataTables.mjs",
-        "https://cdn.datatables.net/1.12.1/js/jquery.dataTables.mjs",
-    ),
-]:
-    r = requests.get(url)
-    with open(str(external / name), "wb") as fp:
-        fp.write(r.content)
-
 setup(
     name="itables",
     version=version,
